app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailViewModel.kt

```diff
@@ -29,9 +31,13 @@ import kotlinx.coroutines.withContext
 class EventDetailViewModel @Inject constructor(
     private val savedStateHandle: SavedStateHandle,
     private val repository: EventRepository,
-    private val uriByteConverter: UriByteConverter
+    private val uriByteConverter: UriByteConverter,
+    private val prefs: SharedPreferences
```

Comment:
Here, you're binding a data layer dependency to your presentation layer. Instead, I'd recommend to abstract out this preferences implementation in domain and data and then use the domain interface in your ViewModel